res/database/Default/user/ariksan.xml

```diff
@@ -361,17 +357,18 @@
 						<en>The Tomb of the Pharao</en>
 					</title>
 					<description>
-						<de>${nominativ_${variant}} ${tier_${variant}} und das ${animal2} tauchen in die Welt der Archäologie ein. Achtung kleines ${animal2}, keine alten Flüche wecken!</de>
+						<de>${nominativ_${variant}} ${tier_${variant}} und das ${adjective} ${animal2} tauchen in die Welt der Archäologie ein. Achtung kleines ${animal2}, keine alten Flüche wecken!</de>
 						<en>The ${adjective} ${animal} and the tiny ${animal2} delve into archaeology. Easy, ${animal2}, no ancient curses or mummies will make great roommates!</en>
 					</description>
+					<episodes min="0" max="1" /> <!-- TODO: no enforced pilot for now (should be more variable) -->
```

Comment:
I really don't know, if mixing episode count definitions in parent in children works properly'!

Reply:
The script appears at least. Without that first episode so far. Haven't tested many times enough to ensure the first episode *would* appear sometimes.

Reply:
I'll check once more in the code. A first test looked OK. Possibly, I will downgrade the editor issue to warning - I think at least mixing entries with max values greater than 1 might cause problems (parent wants 5 episodes, e.g. one episode wants 2-4 instances).

Reply:
The "optional"-pattern for pilot episodes is valid and documented. Typically when choosing a subset of episodes the "pilot" is always included unless it is marked as optional (episodes min="0"). I will fix the validation in the editor.